seqr/management/commands/reload_saved_variant_json_tests.py

```diff
@@ -0,0 +1,83 @@
+#-*- coding: utf-8 -*-
+import mock
+
+from django.core.management import call_command
+from django.test import TestCase
+from django.db.models.query_utils import Q
+from seqr.models import Project
+
+PROJECT_NAME = u'1kg project n\u00e5me with uni\u00e7\u00f8de'
+PROJECT_GUID = 'R0001_1kg'
+PROJECT_GUID2 = 'R0003_test'
+SAVED_VARIANT_GUIDS = ['SV0000001_2103343353_r0390_100', 'SV0000002_1248367227_r0390_100']
+FAMILY_ID = '1'
+
+class ReloadSavedVariantJsonTest(TestCase):
+    fixtures = ['users', '1kg_project']
+
+    @mock.patch('logging.getLogger')
+    @mock.patch('seqr.views.utils.variant_utils.update_project_saved_variant_json')
+    def test_with_param_command(self, mock_update_json, mock_getLogger):
+        mock_update_json.return_value = SAVED_VARIANT_GUIDS
+        logger = mock_getLogger.return_value
+
+        # Test with specific projects and a family id.
+        call_command('reload_saved_variant_json',
+                     PROJECT_NAME, PROJECT_GUID2,
+                     '--family-id=1'.format(FAMILY_ID))
```

Comment:
you are calling `format` but you hardcoded the family id. I think the hardcoded param is fine, so just take out the call to `format`

Reply:
Good catch. I will use `format` because I need to re-use the value of `FAMILY_ID`.